templates/portal/home.html.twig

```diff
@@ -657,6 +630,171 @@
+                <div id="amendmentslist" class="card-body p-2 bg-light"></div>
+            </div> #}
+                    <div class="card">
+                        <header class="card-header p-1 bg-dark text-light">{{ 'Lab Results' | xlt }}</header>
+                        <div id="labresults" class="card-body p-2 bg-light"></div>
+                    </div>
+                </div>
+            </div>
+        </div>
+
+        <div class="card collapse overflow-auto" data-parent="#cardgroup" id="reportcard">
+            <header class="card-header p-1 bg-dark text-light h3">{{ 'Custom Itemised Report' | xlt }}</header>
+            <div id="reports" class="card-body"></div>
+        </div>
+
+        <div class="card collapse" data-parent="#cardgroup" id="profilecard">
+            <div id="profilereport" class="card-body p-2 bg-light"></div>
+        </div>
+
+        <div class="collapse" data-parent="#cardgroup" id="secure-msgs-card">
+            <div class="card">
+                <div id="secure-msgs" class="card-body">
+                    <iframe src="{{ web_root | attr }}/portal/messaging/messages.php" class="w-100 vh-100 border-0"></iframe>
+                </div>
+            </div>
+        </div>
+
+        <div class="collapse" data-parent="#cardgroup" id="portal-settings-card">
+            <div class="card d-flex mr-1 mb-1">
+                {# Work in progress #}
+                <div class="card-body row">
+                    <h4 class="card-title"><i class="fa fa-link mx-1"></i>Settings</h4>
+                    <div class="col-12 form-group">
+                        <div class="col">
+                            <div class="input-group">
+                                <label class="m-0" for='my_theme'>Current Theme</label>
+                                <select class='form-control ml-2' id='my_theme'>
+                                    <!-- Options will be dynamically generated by JavaScript -->
+                                </select>
+                                <div class="input-group-append">
+                                    <button class="btn btn-primary" onclick="applyTheme()">Apply</button>
+                                </div>
+                            </div>
+                        </div>
+                        <div class="col mt-3">
+                            <div class="form-group form-check-inline">
+                                <label class="form-check-label" for="autoSave">
+                                    <input type="checkbox" class="form-check-input" id="autoSave"> Auto Save Documents (Coming soon)
+                                </label>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+
+                <script>
+                    const styleArray = {
+                        'default': 'Default',
+                        'dark': 'Dark Mode',
+                        'light': 'Light Mode'
```

Comment:
i would translate/escape the 3 values (not the keys) above via xla (then will be translated and properly escaped when used in line 719)